bzt/engine.py

```diff
@@ -297,7 +298,8 @@ def _shutdown(self):
         :return:
         """
         self.log.info("Shutting down...")
-        self.log.debug("Current stop reason: %s", self.stopping_reason)
+        if self.stopping_reason:
```

Comment:
I don't like this change. I liked explicit information in this point, if we have exception or not.